app/Models/QueryBuilders/SearchFormQueryBuilder.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace App\Models\QueryBuilders;
+
+use ElasticScoutDriverPlus\Builders\QueryBuilderInterface;
+
+final class SearchFormQueryBuilder implements QueryBuilderInterface
+{
+    /**
+     * @var string
+     */
+    private $query;
+
+    public function query(string $query): self
+    {
+        $this->query = $query;
+        return $this;
+    }
+
+    /**
+     * @var boolean
+     */
+    private $isPublic;
```

Comment:
Shouldn't we define all class properties at the top, instead of above each method?

Reply:
Moved all class properties at the top.

Reply:
It is not showing updated. Did you push your changes, @fahad-curriki ?